Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -125,6 +125,80 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    private func prepareCacheWithIncorrectOffsetInHeader() throws -> CacheHeaderHandle {
+        let randomHighValue: UInt64 = 10_1000
```

Comment:
```suggestion
        let randomHighValue: UInt64 = 101_000
```
While we are here, let's fix this and above to be like ^ as I was misreading this as about 10k but it's really about 100k.